lib/internal/fs/rimraf.js

```diff
@@ -0,0 +1,249 @@
+// The code in this file is adapted from the rimraf module.
```

Comment:
I think we should probably just stick the appropriate license header here as well.

Reply:
Dropping the license header into the LICENSE file should be enough I think?

Reply:
works for me :+1: